content/docs/04-clusters/02-data-center/01-maas/10-register-manage-maas-cloud-accounts.md

```diff
@@ -20,14 +20,19 @@ When you install the Private Cloud Gateway (PCG), a cloud account is auto-create
 - An installed PCG. Review [Install and Manage MAAS Gateway](/clusters/data-center/maas/install-manage-maas-pcg) for guidance.
 
 
+
+- For self-hosted Palette, ensure Palette has network access to the MAAS server.
+Refer to [Register a Self-Hosted Palette Maas Cloud Account](#register-a-self-hosted-palette-maas-cloud-account) to learn more.
+
+
 - An active [MAAS API key](https://maas.io/docs/api-authentication-reference) which can be generated in the MAAS web console under **My Preferences** > **API keys**. The following is an example key:
 
   ``APn53wz232ZwBMxDp5:MHZIbUp3e4DJTjZEKg:mdEv33WAG536MhNC8mIywNLtjcDTnFAQ``
 
  For details, refer to the MAAS document on [how to add an API key](https://maas.io/docs/how-to-manage-user-accounts#heading--api-key).
 
 
```

Comment:
This could be confusing. A user might ask "How do I know if I need a PCG or not. How are we helping the user understand that in this document? 

Also, because the steps will be very similar, we can improve the reader experience by using Tabs. With PCG, and Without PCG.

Reply:
Fixed this according to what we discussed.